variables-iam.tf

```diff
@@ -164,6 +184,8 @@ variable "create_iam_worker_policy" {
   }
 }
 
+# Tagging
+
 variable "create_iam_tag_namespace" {
```

Comment:
ideally this is is what you do in the external iam module

Reply:
Agreed - I've left disabled (default) in practice and created separately for reuse. We could move out of here at some point.